vcl/internal/vcl_atomic_count_sync.h

```diff
@@ -44,9 +44,9 @@ class vcl_atomic_count
   operator long() const { return __sync_fetch_and_add(&value_, 0); }
 
 private:
-  vcl_atomic_count(const vcl_atomic_count &);
+  vcl_atomic_count(const vcl_atomic_count &) = delete;
```

Comment:
Removal of corresponding implementations should be part of this commit.

Reply:
There is no implementation for these private copy constructors since they were never allowed to be used.